ext/case_transform/src/lib.rs

```diff
@@ -2,80 +2,124 @@
 extern crate ruru;
 extern crate inflector;
 
-// // dash: kebab-case
-use inflector::cases::kebabcase::to_kebab_case;
-// // underscore: snake_case
-use inflector::cases::snakecase::to_snake_case;
-// // camel_lower: camelCase
-use inflector::cases::camelcase::to_camel_case;
-// // camel: ClassCase (PascalCase)
-use inflector::cases::classcase::to_class_case;
-
-use ruru::{Class, Object, RString, Hash, Array, Symbol, AnyObject, VM};
-use ruru::types::ValueType;
+use inflector::cases::{camelcase, classcase, kebabcase, snakecase};
 
-class!(CaseTransform);
```

Comment:
why was this removed? I saw it one one of the example repos, though I don't know what its purpose was